docs/using.rst

```diff
@@ -507,8 +507,9 @@ The following files are available:
   <http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_certificate_key>`_.
 
 ``fullchain.pem``
-  All certificates, **including** server certificate. This is
-  concatenation of ``cert.pem`` and ``chain.pem``.
```

Comment:
I think the concatenation statement was correct, right? Did you remove it because you thought it was confusing?